initial_pos : 0
boolean : True
step : 0.01
max_value : 1
min_value : -1
lon_1 : 10
lon_2 : 15
function : "trg"